roulette/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "roulette",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reach/dialog": "^0.17.0",
+    "@reach/listbox": "^0.17.0",
+    "@reach/menu-button": "^0.17.0",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^12.1.1",
+    "@testing-library/user-event": "^12.5.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "eslint": "eslint .",
+    "eslint:fix": "eslint --fix",
+    "prettier": "prettier --check .",
+    "prettier:fix": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}'"
```

Comment:
You're missing "lint" and "lint:fix" commands to check both ESLint & Prettier and check & auto fix both ESLint & Prettier respectively.